01_getting_started/hello_world.py

```diff
@@ -86,16 +83,16 @@ def main():
 # ## What just happened?
 #
 # When we called `.remote` on `f`, the function was executed
-# **in the cloud**, on Modal's infrastructure, not locally on our computer.
+# _in the cloud_, on Modal's infrastructure, not locally on our computer.
 #
 # In short, we took the function `f`, put it inside a container,
 # sent it the inputs, and streamed back the logs and outputs.
 #
 # ## But why does this matter?
 #
-# Try doing one of these things next to start seeing the full power of Modal!
+# Try doing one of these things next to see the full power of Modal.
 #
-# ### You can change the code and run it again
+# ### Change the code and run it again
```

Comment:
I intentionally changed these to be more active and to map onto the question in the section title above: "Why does this matter?".

It matters _because_ you can change the code and run it again -- more quickly and easily than another platform.

It matters _because_ you can map over more data. It matters _because_ you can run something more interesting.

Reply:
<img width="823" alt="image" src="https://github.com/modal-labs/modal-examples/assets/7550632/934554ba-de22-4180-8a9f-f00cfdc44cbc">

"Try doing one of these things next!" ->
"You can change the code"